# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/cracklib/fpic.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nru cracklib,2.7-orig/cracklib/Makefile cracklib,2.7/cracklib/Makefile
--- cracklib,2.7-orig/cracklib/Makefile	2004-11-06 09:48:53.010325096 +0100
+++ cracklib,2.7/cracklib/Makefile	2004-11-06 09:49:08.042039928 +0100
@@ -8,7 +8,7 @@
 
 LIB=	libcrack.a
 OBJ=	fascist.o packlib.o rules.o stringlib.o
-CFLAGS= -O -I../cracklib -DIN_CRACKLIB
+CFLAGS= -O -I../cracklib -DIN_CRACKLIB -fPIC
 
 $(LIB):	$(OBJ)
 	ar rv $(LIB) $?
